fhir-search/src/test/java/com/ibm/fhir/search/sort/test/SortParameterParseTest.java

```diff
@@ -83,76 +116,71 @@ public void testUnknownSortParm_strict() throws Exception {
 
     /**
      * Tests a valid sort with: asc modifier, and a valid parameter value.
-     * 
-     * @throws Exception
      */
     @Test
     public void testValidSortParm() throws Exception {
         Map<String, List<String>> queryParameters = new HashMap<>();
         FHIRSearchContext searchContext;
         Class<Patient> resourceType = Patient.class;
-        SortDirection direction = SortDirection.ASCENDING;
         String sortParmName = "birthdate";
-        String queryString = "&_sort:" + direction.value() + "=" + sortParmName;
+        String queryString = "&_sort=" + sortParmName;
 
-        queryParameters.put("_sort:" + direction.value(), Collections.singletonList(sortParmName));
+        queryParameters.put("_sort", Collections.singletonList(sortParmName));
         searchContext = SearchUtil.parseQueryParameters(resourceType, queryParameters);
         assertNotNull(searchContext);
 
         // Do sort parameter validation
         assertNotNull(searchContext.getSortParameters());
         assertEquals(1, searchContext.getSortParameters().size());
         SortParameter sortParm = searchContext.getSortParameters().get(0);
-        assertEquals(sortParmName, sortParm.getName());
-        assertEquals(direction, sortParm.getDirection());
+        assertEquals(sortParmName, sortParm.getCode());
```

Comment:
(1) not sure why change Name to Code for search parameter.  
(2) seems in the test codes, Name is still used, e.g, sortParmName instead of sortParmCode.

Reply:
it should always be code, and not name... I'll change to code in these cases

Reply:
fixed in the next commit... (2) ... (1)